cfgov/v1/models/molecules.py

```diff
@@ -12,7 +12,7 @@ def isRequired(field_name):
 
 
 class HalfWidthLinkBlob(blocks.StructBlock):
-    heading = blocks.CharBlock(max_length=100, required=True)
+    heading = blocks.CharBlock(max_length=100, required=False)
```

Comment:
i'd rather not, at this time since it may not be the case in the future and i would have to deal with data migrations with that change.